src/corecel/random/data/RanluxppRngData.hh

```diff
@@ -0,0 +1,120 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+//---------------------------------------------------------------------------//
+/*!
+ * \file corecel/random/data/RanluxppRngData.hh
+ *
+ * Original source:
+ * https://github.com/apt-sim/AdePT/blob/master/include/AdePT/copcore/Ranluxpp.h
+ */
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <algorithm>
+
+#include "corecel/Assert.hh"
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/cont/Range.hh"
+#include "corecel/data/Collection.hh"
+#include "corecel/random/data/RanluxppTypes.hh"
+#include "corecel/random/data/detail/RanluxppImpl.hh"
+
+namespace celeritas
+{
+
+//---------------------------------------------------------------------------//
+/*!
+ * Persistent data for the Ranluxpp random number generator.
+ */
+template<Ownership W, MemSpace M>
+struct RanluxppRngParamsData
+{
+    //// DATA ////
+    RanluxppArray9 kA_2048;
+    int kMaxPos;
```

Comment:
The use of `k` meaning "constant" is the old Hungarian notation and not something we do in Celeritas (and besides they're no longer constant. The names are also not very meaningful, so I would be happy to see them changed.